oauth2_http/java/com/google/auth/oauth2/ServiceAccountJwtAccessCredentials.java

```diff
@@ -332,35 +331,17 @@ public boolean hasRequestMetadataOnly() {
     return true;
   }
 
-  /**
-   * Self signed JWT uses uri as audience, which should have the "https://{host}/" format. For
```

Comment:
why this no longer needed?

Reply:
This was added by me in phrase 1, so I could create a ServiceAccountJwtAccessCredentials inside service account credentials to do the actual self signed jwt work. In this PR I switched to use JwtCredentials instead, so I reverted the changes made to ServiceAccountJwtAccessCredentials.